test/PlayerTests.cpp

```diff
@@ -0,0 +1,31 @@
+#include <gtest/gtest.h>
+#include "Player.hpp"
+#include "Property.hpp"
+
+struct PlayerTests : public ::testing::Test
+{
+    Player player{"red"};
+    Property vienna{"Vienna"};
+
+
+};
+
+TEST_F(PlayerTests, check_if_player)
+{
+    ASSERT_EQ("red", player.getColor());
+    ASSERT_EQ(3000,  player.amountOfMoney());
+    ASSERT_EQ(0,player.getLocation());
+    ASSERT_FALSE(player.ownsProperty());
+}
+
+TEST_F(PlayerTests, check_if_location_sets_properly)
```

Comment:
The words like properly, accordingly should be avoided in test names. For one person properly means something else than to another one. If should be clearly stated - setLocationTo12ShouldMoveThePlayerToField12 or something like this.